tcms/tests/__init__.py

```diff
@@ -227,3 +227,23 @@ def setUpTestData(cls):
         cls.execution_4 = executions[3]
         cls.execution_5 = executions[4]
         cls.execution_6 = executions[5]
+
+
+class CreateTestUsers(LoggedInTestCase):
+    """Creates test users with or without requested permissions """
+
+    @classmethod
+    def createTestUserWithNoPermissionExceptTested(cls, tested_permission):
+
+        for perm in Permission.objects.all():
+            remove_perm_from_user(cls.tester, perm)
+
+        user_should_have_perm(cls.tester, tested_permission)
+
+    @classmethod
+    def createTestUserWithAllPermissionExceptTested(cls, tested_permission):
+
+        for perm in cls.all_perms:
```

Comment:
what is the reason we iterate over `cls.all_perms` here, but over `Permission.objects.all()` in the other method?